documentation/asciidoc/computers/configuration/uart.adoc

```diff
@@ -4,6 +4,30 @@ There are two types of UART available on the Raspberry Pi -  http://infocenter.a
 
 All UARTs on the Raspberry Pi are 3.3V only - damage will occur if they are connected to 5V systems. An adaptor can be used to connect to 5V systems. Alternatively, low-cost USB to 3.3V serial adaptors are available from various third parties.
 
+=== Raspberry Pi 5
```

Comment:
The "additional" comment makes more sense after the Pi 0-3 section (and after Pi 4 is more logical), and we should mention that the mini-UART has gone.

Reply:
And this comment?

Reply:
I moved the section and added a note about the mini UART.